### ISSUE TITLE
Unhide dist-modern in VSCode file tree

### ISSUE THREAD

**Issue by martenbjork** [Shopify/polaris-tokens#109]
`.vscode/settings.json` hides `dist-modern` from the file tree. This caused me a real headache when trying to see the results of my adjustments. Is there a reason for this?

  ```
"files.exclude": {
    "**/.DS_Store": true,
    "**/.git": true,
    "**/node_modules": true,
    ".{bundle,dev,shadowenv.d}": true,
    "{tmp,dist-modern}/": true
  },

**Comment by kaelig** [comment]
That's a good point, I'll let @danrosenthal and @BPScott weigh in on this, as they recently set this new structure up.

**Comment by BPScott** [comment]
My thinking is that you don't really care about generated contents. This is a little bit rubbish right now but medium term idea is that we provide a proper documentation page for the new colors, which you can use to see what changes have taken effect, so you can see these changes in your browser rather than looking at generated content.

**Comment by kaelig** [comment]
The changes that @martenbjork is working on (adding metadata fields) have an impact on the shape of the output – this raises the need for thorough API docs (not just usage docs), if we want him to avoid having to look at the filesystem.

**Comment by martenbjork** [comment]
I think leading people to the docs is great, but hiding folders from
developers is a strange way to do that. In my case, I was digging around in
the pipeline, trying to understand how things work. And despite seeing many
references to /dist-modern, that file didn't show up in the file tree. I
thought I was doing something wrong.

In my case, I wanted to get the raw json output of the pipeline, not just a
cleaned up version of it.

TLDR: It's hard to predict what devs might want to get out of the code 🙂

**Comment by danrosenthal** [comment]
Seems fairly risk-free to just expose this in vscode. I'm in favor of doing it if it helps our consumers build tooling around this work.

**Comment by BPScott** [comment]
Yeah it's fine to do for now :) I generally get annoyed by searching for things in a project and having to sift through twice the results I was hoping for because transpiled output is included in my search results, which is why i lean towards hiding dist folders.

**Comment by kaelig** [comment]
I have empathy towards @martenbjork and also understand @BPScott's point of view.

Here's my 2 cents: since the main point of tweaking formats/transforms is to obtain a very specific output (we're basically working on the transpiler itself), this is one of the rare cases where giving visibility to the dist folder (and versioning the output as well) makes sense. Having a very thorough and hot-reloaded documentation might replace this need, but looking at the output itself carries a higher confidence IMHO.

**Comment by martenbjork** [comment]
Thanks for the input, everyone! I've created a PR to make that directory visible. (Assuming that I've read the conclusion of this thread correctly 🙂)
